plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/dataimport/CommonElementImporter.java

```diff
@@ -882,6 +882,7 @@ protected static String fullyUnEscapeValue(final String value) {
 		escapedValue = escapedValue.replace("&apos;", "\'"); //$NON-NLS-1$ //$NON-NLS-2$
 		escapedValue = escapedValue.replace("&#10;", "\n"); //$NON-NLS-1$ //$NON-NLS-2$
 		escapedValue = escapedValue.replace("&#9;", "\t"); //$NON-NLS-1$ //$NON-NLS-2$
+		escapedValue = escapedValue.replace("\r\n", "\n"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
I think it is better to only check for \r because the old escaping could resulted in `\r&#10;

Reply:
at this point in the unescape-sequence `&#10;` has already be replaced back to `\n` - everything should already be back to `\r\n` here

Reply:
ah missed that. Thx.